modules/getting-started/proc-audit-log-file-rotation.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assembly:
+// assembly-audit-log.adoc
+
+[id="proc-audit-log-file-rotation_{context}"]
+= Configure Audit Log file rotation
+
+You can enable file rotation for audit logs in {product}, and configure the file log location, name format, log file size, and archiving.
+
+.Prerequisite
+
+* You are an administrator of {product-short}.
+
+.Procedure
+
+. Enable audit logging to a rotating file:
++
+[source, yaml]
+----
+auditLog:
+  rotateFile:
+    enabled: true
+----
++
+. Customize the log file location:
++
+[source, yaml]
+----
+auditLog:
+  rotateFile:
+    logFileDirPath: /custom-path <1>
+----
+<1> An absolute path is recommended.
```

Comment:
As per IBMSG, avoid word recommended. See the screenshot below:
![image](https://github.com/user-attachments/assets/d866f77a-1206-4862-9ed3-8ffd074d6e4d)

```suggestion
<1> Absolute path to the log file.
```